exercises/pythagorean-triplet/pythagorean-triplet_spec.lua

```diff
@@ -18,25 +18,57 @@ describe('pythagorean-triplet', function()
     end)
   end)
```

Comment:
How about removing `is_triplet`? It's no longer reflected in https://github.com/exercism/problem-specifications/blob/master/exercises/pythagorean-triplet/canonical-data.json and it's generally favorable to avoid tests that prescribe a particular solution.